### ISSUE TITLE
Options don't work when defined in TS file

### ISSUE THREAD

**Issue by MusicDev33** [drozhzhin-n-e/ng2-tooltip-directive#65]
Adding options in the `.ts` file doesn't seem to work. I set them in the component's TypeScript file, then use them in the HTML:
```TypeScript
ttOptions = { 'hide-delay': 300 }
```
And in the HTML:
```HTML
<span [options] = "ttOptions">
```
And I still have to set them manually in the HTML because options aren't binding somehow (no errors get thrown).

**Comment by cosgab** [comment]
In the following option object 'placement' and 'theme' work, the other don't. 

  myOptions = {
    'show-delay': 4000,
    'hide-delay': 1000,
    'placement': 'left',
    'theme': 'light',
    'tooltip-class': 'tooltip'
  };

**Comment by drozhzhin-n-e** [comment]
@MusicDev33 @cosgab  fixed in version 2.2.9
